core/GlobalState.cc

```diff
@@ -938,7 +965,7 @@ NameRef GlobalState::getNameUnique(UniqueNameKind uniqueNameKind, NameRef origin
         bucketId = (bucketId + probeCount) & mask;
         probeCount++;
     }
-    Exception::raise("should never happen");
```

Comment:
This changed because we now want to speculatively find a mangled name if one has been created for the purposes of `lookupSymbolWithFlags`: we might need to walk through all mangled names to find the relevant one, which means we can use this function to find the next mangled name if it exists.

Reply:
let's rename it to `lookup`. `get` stands for `fail if you can't` while `lookup` mail return `not found`.